modules/EvseSlac/manifest.yaml

```diff
@@ -46,6 +46,34 @@ provides:
           It is better to wait for the matching to be done.
         type: boolean
         default: false
+      do_chip_reset:
+        description: Perform a chip reset after setting NMK using the RS_DEV.REQ Vendor MME Extension (QCA7k)
+        type: boolean
+        default: false
+      chip_reset_delay_ms:
+        description: Delay between SET_KEY.CNF and RS_DEV.REQ
```

Comment:
Just a note: i've noticed that the QCA7K does not always send a SET_KEY.CNF (or the host has a chance to receive it). Unfortunately i never had the time to investigate this further. AFAIR this was the case for firmware before 3.0.0 and non-fast SLAC configuration which does a real reset after CM_SET_KEY, but i'm not totally sure.

This is just a warning, not a request to change.

Reply:
Interesting. The non-fast SLAC config, does that mean the QCA resets itself when setting a key? I think we always used a config where the QCA does not automatically reset when the key is changed, that is why we do a reset here manually afterwards. I have not seen the issue of missing CNF in our setup, but it would make sense that e.g. the QCA itself resets before the CNF packet is really transmitted to the host

Reply:
So maybe using fast config option is preffered as we have more control over when (or if at all) we need to reset

Reply:
> Interesting. The non-fast SLAC config, does that mean the QCA resets itself when setting a key? I think we always used a config where the QCA does not automatically reset when the key is changed, that is why we do a reset here manually afterwards. I have not seen the issue of missing CNF in our setup, but it would make sense that e.g. the QCA itself resets before the CNF packet is really transmitted to the host

From my understanding the behavior in this case is the following:
- EVSE host send CM_SET_KEY.REQ
- QCA7000 stores the the NMK in the internal flash
- QCA7000 send CM_SET_KEY.CNF
- QCA7000 triggers a reset, which clears the messages buffers

Yes, i agree this process is not recommend for production because of the flash wearout.

Reply:
The fast way also writes it to flash memory, after power cycle the key is still there. I think the chip was designed for home usage where the PLC modems are paired a few times per lifetime. It was not really designed for changing a key very often.

Reply:
Okay, the other advantage of fast way was avoiding the reset which takes ~ 7 second before SLAC is operational again.